Tank.java

```diff
@@ -1 +1,459 @@
-please ignore
\ No newline at end of file
+import java.awt.*;
+import java.awt.event.*;
+import java.util.*;
+
+public class Tank {
```

Comment:
Tanko klasė daro per daug. Tankų klasė yra atsakinga už poziciją ir už daug kitų dalykų.
Galima būtų sukurti atskiras klases kiekvienai iš pareigų.
Pvz. galima būtų sukurti atskira klasę Position, kuri valdytų x ir y koordinates.

Pritaikyti Single Responsibility Principle 
Single Responsibility Principle states that a class should have only one reason to change, which means that it should have only one responsibility or concern.